i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/traits/patch-trait.md

```diff
@@ -354,7 +378,7 @@ spec:
         		// +patchKey=name
         		containers: [{
         			name: context.name
-        			// +patchKey=name
+        			// +patchStrategy=retainKeys
```

Comment:
typo?

Reply:
We should use `retainKeys` here, the original example has some problems.